### PR TITLE
style: added package json sorting prettier plugin

### PR DESCRIPTION
**Pull request by justinfaynhan**
Added package json sorting plugin alongside some lint configuration and script cleanups

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Logs
```
Comment (justinfaynhan) [inline .eslintignore:1]: Separating eslintignore from gitignore for  more explicitness and specific configurations

File: .prettierignore [hunk 2]
```diff
@@ -1,6 +1,51 @@
-# Add files here to ignore them from prettier formatting
```
Comment (justinfaynhan) [inline .prettierignore:1]: Separating prettierignore from gitignore for  more explicitness and specific configurations

File: package.json [hunk 3]
```diff
@@ -67,6 +67,7 @@
     "patch-package": "^6.4.7",
     "postinstall-postinstall": "^2.1.0",
     "prettier": "2.3.2",
+    "prettier-plugin-packagejson": "^2.2.17",
```
Comment (justinfaynhan) [inline package.json:70]: Prettier plugin to execute package json sorting whenever prettier is executed

File: packages/float-input/package.json [hunk 4]
```diff
@@ -1,17 +1,9 @@
 {
```
Comment (justinfaynhan) [inline packages/float-input/package.json:1]: Existing lint errors that got picked up with the new changes

**Review by lukebennett88 — APPROVED** [review]

**Pull request merged**